### ISSUE TITLE
Cannot Backup Chat To Google Drive 

### ISSUE THREAD

**Issue by TashonBraganza** [Schwartzblat/WhatsAppPatcher#69]
Cannot Backup chats to Google drive ,as it is asking for permission and give error when trying to sign in
![Screenshot_20231213-220245](https://github.com/Schwartzblat/WhatsAppPatcher/assets/58265596/2784229e-5c5f-4a58-b712-60d416555d8c)
![Screenshot_20231213-220321](https://github.com/Schwartzblat/WhatsAppPatcher/assets/58265596/a137bc89-30a6-4fc7-b227-70e354e6d391)

**Comment by vinisantt** [comment]
I got the same error, but during installation, so I didn't proceed.

**Comment by Schwartzblat** [comment]
Does it still happening?

**Comment by TashonBraganza** [comment]
Yes it still isn't backing up, it is showing as ' unable to access account '

**Comment by KurtzPT** [comment]
@Schwartzblat just installed the patched apk and I'm also having the same issue.

**Comment by Schwartzblat** [comment]
Okay I will look into it.

**Comment by theSoberSobber** [comment]
> Okay I will look into it.

Pretty sure they detect signature or perform some other integrity checks for it, use something like [this](https://github.com/L-JINBIN/ApkSignatureKillerEx), maybe?
Good work btw, make a dc server maybe? I would love to contribute!

**Comment by S0PEX** [comment]
Are there any updates on this issue?

**Comment by Appmeurtre** [comment]
Issue continue, does anyone able to solve this?

**Comment by Gustavoppw** [comment]
I have the same exact issue, can't use it without it

**Comment by Schwartzblat** [comment]
I will look into it.

**Comment by Schwartzblat** [comment]
Okay, it's too much for me right now, it's a google security mechanism called oauth2 and they check the signature of the package from other process.
I am not sure if I can bypass it because of my limited free time.
I will keep this issue open but I am not going to invest efforts in it soon.
